### PR TITLE
Support for the deploy Github Action

### PR DESCRIPTION
**Pull request by D3473R**
With this PR you only need to add a Readme or Wiki Entry which could look like this:

1. Generate a set of Ghost Admin API credentials, by configuring a new Custom Integration in Ghost Admin&raquo;Integrations. 
2. On GitHub, navigate to your theme repository&raquo;Settings&raquo;Secrets. Create a secret called `GHOST_ADMIN_API_URL` containing the API URL and another called `GHOST_ADMIN_API_KEY` containing the Admin API Key. Both must be copied exactly from Ghost Admin&raquo;Integrations.
3. Create a file `.github/workflows/deploy-theme.yml` with the contents:
```yaml
name: Deploy Theme
on: [push]

jobs:
  deploy:
    runs-on: ubuntu-18.04
    steps:
      - uses: actions/checkout@master
      - uses: actions/setup-node@v1
        with:
          node-version: '10.x'
      - run: npm install
        working-directory: ./src
      - name: Build and deploy the theme
        working-directory: ./src
        run: npm run deploy
        env:
          GHOST_ADMIN_API_URL: ${{ secrets.GHOST_ADMIN_API_URL }}
          GHOST_ADMIN_API_KEY: ${{ secrets.GHOST_ADMIN_API_KEY }}
```

Manual deployment is also possible.
1. Copy `.env.example` to `.env`
2. Enter your `GHOST_ADMIN_API_URL` and `GHOST_ADMIN_API_KEY` in the `.env` file (Not in the `.env.example` file!!! Because this file is tracked with git and your secrets would be uploaded to your git repo)
3. Run `npm run deploy` in the `src` directory
Now the theme gets build and deployed to your ghost instance.

### CODE REVIEW

File: src/package.json [hunk 1]
```diff
@@ -31,8 +33,10 @@
     "vue-template-compiler": "^2.6.10"
   },
   "dependencies": {
+    "@tryghost/admin-api": "^1.0.1",
```
Comment (eddiesigner) [inline src/package.json]: Can you please add this as a dev-dependency?

File: src/package.json [hunk 2]
```diff
@@ -31,8 +33,10 @@
...
     "aos": "^2.3.4",
     "css-vars-ponyfill": "^2.1.2",
+    "dotenv": "^8.2.0",
```
Comment (eddiesigner) [inline src/package.json]: Can you please add this as a dev-dependency?

**Comment by eddiesigner** [comment]
This is really nice, thanks for your contribution! 😄

I just wrote a couple of comments, after that I will merge it and then add a new entry to the Wiki.

**Comment by D3473R** [comment]
> This is really nice, thanks for your contribution! 😄
> 
> I just wrote a couple of comments, after that I will merge it and then add a new entry to the Wiki.

Ahhh sure :D, I overlooked that.

**Pull request merged**